src/Gallery.js

```diff
@@ -19,13 +19,17 @@ export default class Gallery extends PureComponent {
         onLongPress: PropTypes.func,
         removeClippedSubviews: PropTypes.bool,
         imageComponent: PropTypes.func,
-        errorComponent: PropTypes.func
+        errorComponent: PropTypes.func,
+        flatListProps: PropTypes.object,
     };
 
     static defaultProps = {
         removeClippedSubviews: true,
         imageComponent: undefined,
-        scrollViewStyle: {}
+        scrollViewStyle: {},
+        flatListProps: {
+            windowSize: 3,
```

Comment:
Maybe we should set this `windowSize` value directly where we render `Flatlist` to make sure this isn't overriden when the users use `flatListProps` prop.
Or, we would have to merge the default value and the user's one.

Reply:
Yup. Makes sense. The soonest I'll be able to fix this is on Monday.